adgorithmics-inc/cinnamon-javascript-sdk
# Add password reset

## Unit Test Coverage / Review Requirements
* Are the changes in this PR covered by unit tests?
  *
* Does this PR create or modify code without tests?  If so, please explain.
  *

## How to Test
 * run requestPasswordReset
   - verify works
 * run resetPassword
   - verify works

## Changelog
* Added requestPasswordReset and resetPassword mutations

## Notes
